release.py

```diff
@@ -0,0 +1,694 @@
+# Copyright 2025 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Release automation script."""
+
+from __future__ import annotations
+
+import argparse
+import logging
+import os
+import re
+import subprocess
+from datetime import datetime
+from pathlib import Path
```

Comment:
[Our style](https://github.com/canonical/operator/blob/main/STYLE.md#import-modules-not-objects) is `import package`:
```suggestion
import datetime
import pathlib
```